src/styles/main.scss

```diff
@@ -1,3 +1,83 @@
-body {
+$cell-size: 100px;
+$gap-size: 1px;
+$padding: 25px 0 0;
+$sheet-with: $cell-size * 7 + $gap-size * 6;
+
+$days-in-week: mon 1, tue 2, wed 3, thu 4, fri 5, sat 6, sun 7,;
+$days-in-month: 28, 29, 30, 31;
+
+$number-font: "Arial", sans-serif;
+$number-font-size: ($cell-size * 0.3);
+$number-font-position: ($cell-size - $number-font-size) / 2;
+
+* {
   margin: 0;
+  padding: 0;
+  animation-duration: 10s;
```

Comment:
It is recommended to use * only for box-sizing as other properties can affect the performance.
Btw, why do you need animation-duration? You don't have any animations in your code